app/scripts/modules/openstack/src/serverGroup/details/serverGroupDetails.html

```diff
@@ -166,7 +166,6 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
         <dd ng-repeat-end>{{value}}</dd>
       </dl>
     </collapsible-section>
-  </div>
```

Comment:
suspicious

Reply:
suspicious? What do you mean?

Reply:
Just wanted to make sure prettier wasn't removing a closing tag erroneously and possibly breaking the layout. It wasn't obvious to me if this was just an extraneous tag or not.

Reply:
ah I see.  This commit wasn't prettier, this was all by hand to fix the HTML spec issues prettier pointed out

Reply:
Ah! Sorry to waste yer time